lib/internal/debugger/inspect.js

```diff
@@ -45,7 +45,7 @@ const debuglog = util.debuglog('inspect');
 
 const { ERR_DEBUGGER_STARTUP_ERROR } = require('internal/errors').codes;
 
-async function portIsFree(host, port, timeout = 9999) {
+async function portIsFree(host, port, timeout = 3000) {
```

Comment:
the pr title says raise, but this does the opposite.

Reply:
That's from https://github.com/nodejs/node/pull/44359, which would be rebased out here after it lands ;)